test/e2e/test-data/sample-deploy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
```

Comment:
I don't think this is used?

Reply:
No, but I meant to have a sample of what the deployment looks like. It's totally fine to delete it